frontend/src/Components/LayoutsWithNavbar.js

```diff
@@ -5,16 +5,18 @@ import { GlobalContext } from '../GlobalContext';
 import NavBar from './NavBar';
 
 export default function LayoutsWithNavbar() {
-  const {setMatrixUserId} = useContext(GlobalContext);
+  const {setMatrixUserId, setGithubUserId} = useContext(GlobalContext);
   const navigate = useNavigate();
 
   useEffect( () => {
       const fetchCurrentUser = async () => {
         const resp = await axios.get('api/current-user');
         console.log(resp.data);
         if(resp.data.matrix_user_id !== null){
-            const user_id = resp.data.matrix_user_id;
-            setMatrixUserId(user_id);
+            const matrix_user_id = resp.data.matrix_user_id;
+            const github_user_id = resp.data.github_user_id ? resp.data.github_user_id : "";
+            setMatrixUserId(matrix_user_id);
+            setGithubUserId(github_user_id);
```

Comment:
mixture of casing, would prefer camelCase

```suggestion
            const matrixUseId = resp.data.matrix_user_id;
            const githubUserId = resp.data.github_user_id ? resp.data.github_user_id : "";
            setMatrixUserId(matrixUseId);
            setGithubUserId(githubUserId);
```

Reply:
Done!

https://github.com/MatrixDoctors/matrix-cerberus/blob/398e3af73cfb6f7867b9553e7d9de7d8d8c86300/frontend/src/Components/LayoutsWithNavbar.js#L11-L28